src/org/labkey/test/components/ui/entities/EntityInsertPanel.java

```diff
@@ -196,6 +234,44 @@ protected boolean isVisible(Locator locator)
         }
     }
 
+    private EntityInsertPanel showGrid()
+    {
+        if (!isGridVisible())
+            elementCache().modeSelectListItem("from Grid")
+                    .waitForElement(this, 2000).click();
+
+        WebDriverWrapper.waitFor(()-> isGridVisible(),
+                "the grid did bot become visible", 2000);
+        return this;
+    }
+
+    private EntityInsertPanel showFileUpload()
+    {
+        if (!isFileUploadVisible())
+            elementCache().modeSelectListItem("from File")
+                    .waitForElement(this, 2000).click();
```

Comment:
Again if I understand the component (please add a screen shot to clarify), this is dangerous because this is clicking a tab to switch panels, and this is being done in a panel. This is a function more appropriate at a page.

Reply:
The tab-switch is, as I understand, part of the component.  (This is based on my conversations with Cory about testing these in the core-components.view of every labkey folder)  Here it is, in the core-components view: 
![image](https://user-images.githubusercontent.com/16809856/110984332-d8d51480-831f-11eb-89c4-2356f58b9ba6.png)

If this is a hill you're willing to die on, fine. we can scrap this branch and I'll write pages in Biologics that don't expect the component to do this kind of thing